drivers/copy/copy_test.go

```diff
@@ -83,6 +84,13 @@ func randomMode(baseMode int) os.FileMode {
 
 func populateSrcDir(t *testing.T, srcDir string, remainingDepth int) {
 	if remainingDepth == 0 {
+		socketName := filepath.Join(srcDir, "srcsocket")
+		s, err := net.Listen("unix", socketName)
+		assert.NilError(t, err)
+		if unixListener, ok := s.(*net.UnixListener); ok {
+			unixListener.SetUnlinkOnClose(false)
+		}
+		s.Close()
```

Comment:
Assert that the file still exists afterwards? Otherwise the test code and the test subject are exactly the same and might have the same bug.

Reply:
This comment still seems to be open, we should ensure the file exists after the copy.

Reply:
I think the file is ensured to exist by way of https://github.com/containers/storage/pull/2117/files#diff-e80de23fd41197b1acaf1ebc6ec79d3df1d7f56704005df462246eef120b17e3L39-L74